Update mount parameters (#4582)
ArduPilot/ardupilot_wiki: common/source/docs/common-camera-gimbal.rst

.. figure:: ../../../images/MPCameraAndGimbalSetupScreen.jpg
   :target: ../_images/MPCameraAndGimbalSetupScreen.jpg

   Mission Planner: Camera and Gimbal Setup Screen

If you wish to adjust the gimbal pan, tilt, or roll while flying, you can

> set the RC controls for pan, tilt, or roll using :ref:`RCx_OPTION<RC1_OPTION>`.

Reviewer: ```suggestion
set the RC controls for roll, tilt, or pan using ``RCx_OPTION`` 212 (Mount1 Roll), 213 (Mount1 Pitch), 214 (Mount1 Yaw) for the first mount, or 215-217 for the second mount, respectively.
```

If a retractable mount is employed, the overall mount may be deployed or retracted using an output assigned with ``SERVOx_FUNCTION`` set to "MountOpen". This will be automatically controlled by the autopilot as if it were landing gear (see :ref:`common-landing-gear`), or by pilot using an rc channel whose ``RCx_OPTION`` is set to "Landing Gear".

> The default targeting mode for the camera/gimbal is set by the :ref:`MNT1_DEFLT_MODE<MNT1_DEFLT_MODE>` parameter.

Reviewer: ```suggestion
The default targeting mode for the first camera/gimbal is set by the :ref:`MNT1_DEFLT_MODE<MNT1_DEFLT_MODE>` parameter, while ref:`MNT2_DEFLT_MODE<MNT2_DEFLT_MODE>` is used for the second mount, if used.
```

> The direction the axes are set for the NEUTRAL and RETRACTED modes are set by:

Reviewer: ```suggestion
The direction the axes are set for the NEUTRAL and RETRACTED modes are set by (shown for the first mount):
```
